### ISSUE TITLE
Bir compiler: error compiling :cl+ssl (COMPILE-AST #<CLASP-CLEAVIR-AST:DEFCALLBACK-AST>)

### ISSUE THREAD

**Issue by kpoeck** [clasp-developers/clasp#1074]
```lisp
...
(ql:quickload :cl+ssl :verbose t)
...
;   (DEFUN RELOAD ...)
; caught ERROR:
;   No applicable methods in required group PRIMARY for generic function COMPILE-AST
;   Given arguments: (#<CLASP-CLEAVIR-AST:DEFCALLBACK-AST>
;                     #<CLEAVIR-AST-TO-BIR:INSERTER>
;                     #<CLASP-CLEAVIR::CLASP-64BIT>)
; 
; Condition of type: SIMPLE-PROGRAM-ERROR
; Add support for source-file-info for (#<SOURCE-POS-INFO :fileId 492 :filepos 32956 :lineno 935 :column 0> . #<SOURCE-POS-INFO :fileId 492 :filepos 33211 :lineno 940 :column 32>)
; 
; Available restarts:
; (use :r1 to invoke restart 1, etc.)
; 
; 1. (RETRY) Retry compiling #<CL-SOURCE-FILE "cl+ssl" "src" "ffi">.
; 2. (ACCEPT) Continue, treating compiling #<CL-SOURCE-FILE "cl+ssl" "src" "ffi"> as having been successful.
; 3. (RETRY) Retry ASDF operation.
; 4. (CLEAR-CONFIGURATION-AND-RETRY) Retry ASDF operation after resetting the configuration.
; 5. (RETRY) Retry ASDF operation.
; 6. (CLEAR-CONFIGURATION-AND-RETRY) Retry ASDF operation after resetting the configuration.
; 7. (ABORT) Give up on "cl+ssl"
; 8. (RESTART-TOPLEVEL) Go back to Top-Level REPL.
; 
l_SIGNAL-SIMPLE-ERROR^CORE^FN^^
````

**Comment by kpoeck** [comment]
top of backtrace:
```lisp
COMMON-LISP-USER>> (clasp-debug:print-backtrace)

   4: l_NO-REQUIRED-METHOD^CLOS^FN^^
   5: (CLOS::NO-REQUIRED-METHOD #<STANDARD-GENERIC-FUNCTION CLEAVIR-AST-TO-BIR:COMPILE-AST> CLOS::PRIMARY #<CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
   8: ((LAMBDA (&REST CLOS::VASLIST-ARGS)) #<CLASP-CLEAVIR-AST:FOREIGN-CALL-POINTER-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  13: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((PROGN-AST T T))^METHOD^^
  14: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:PROGN-AST T T)) #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  15: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((:AROUND) (AST T T))^METHOD^^
  16: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (:AROUND) (CLEAVIR-AST:AST T T)) #<FUNCTION (METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:PROGN-AST T T))> #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  19: (CLOS::EMF-FROM-CONTF.LAMBDA #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  24: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((PROGN-AST T T))^METHOD^^
  25: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:PROGN-AST T T)) #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  26: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((:AROUND) (AST T T))^METHOD^^
  27: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (:AROUND) (CLEAVIR-AST:AST T T)) #<FUNCTION (METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:PROGN-AST T T))> #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  30: (CLOS::EMF-FROM-CONTF.LAMBDA #<CLEAVIR-AST:PROGN-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  35: l_COMPILE-FUNCTION^CLEAVIR-AST-TO-BIR^((FUNCTION-AST T))^METHOD^^
  36: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-FUNCTION (CLEAVIR-AST:FUNCTION-AST T)) #<CLEAVIR-AST:FUNCTION-AST> #<CLASP-CLEAVIR::CLASP-64BIT>)
  41: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((FUNCTION-AST T T))^METHOD^^
  42: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:FUNCTION-AST T T)) #<CLEAVIR-AST:FUNCTION-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  43: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((:AROUND) (AST T T))^METHOD^^
  44: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (:AROUND) (CLEAVIR-AST:AST T T)) #<FUNCTION (METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:FUNCTION-AST T T))> #<CLEAVIR-AST:FUNCTION-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  47: (CLOS::EMF-FROM-CONTF.LAMBDA #<CLEAVIR-AST:FUNCTION-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  52: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((CALL-AST T T))^METHOD^^
  53: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:CALL-AST T T)) #<CLEAVIR-AST:CALL-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  54: l_COMPILE-AST^CLEAVIR-AST-TO-BIR^((:AROUND) (AST T T))^METHOD^^
  55: ((METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (:AROUND) (CLEAVIR-AST:AST T T)) #<FUNCTION (METHOD CLEAVIR-AST-TO-BIR:COMPILE-AST (CLEAVIR-AST:CALL-AST T T))> #<CLEAVIR-AST:CALL-AST> #<CLEAVIR-AST-TO-BIR:INSERTER> #<CLASP-CLEAVIR::CLASP-64BIT>)
  ````

**Comment by Bike** [comment]
Should be fixed by 4d7a6a031940a62f496378b4ec6b717e5132a9bf, though I think I saw an unrelated error afterward

**Comment by kpoeck** [comment]
Yes, new error now, so closing this and opening a new one
